File: ServerProject/src/main/java/com/example/meditime/model/AiAnalysisResponse.java
package com.example.meditime.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class AiAnalysisResponse {
    private String patterns;
    private String risks;
    private String recommendations;
}

File: Mobile/app/src/main/java/com/example/medicationapp/model/AiAnalysisResponse.kt
package com.example.medicationapp.model

/**
 * AiAnalysisResponse.kt
 *
 * Data class representing the response from the AI analysis engine.
 * This model holds textual data related to the analysis of a client's
 * medication adherence patterns, potential risks, and recommendations.
 **/

 class AiAnalysisResponse {
    internal var patterns: String = ""
    internal var risks: String = ""
    internal var recommendations: String = ""
}